src/quantuminspire/job.py

```diff
@@ -1,27 +1,29 @@
+from typing import Dict, Any
+from collections import OrderedDict
 from coreapi.exceptions import ErrorMessage
 
 
 class QuantumInspireJob:
 
-    def __init__(self, api, job_identifier):
-        """ The QuantumInstpire Job class encapsulates the base job of the API and has
-            methods to check the status and retieve the results from the API.
+    def __init__(self, api: Any, job_identifier: int) -> None:
+        """ The QuantumInspire Job class encapsulates the base job of the API and has
+            methods to check the status and retrieve the results from the API.
 
         Arguments:
             api (QuantumInspireApi): An instance to the API.
```

Comment:
Here is still a docstring typehint. And was there a problem with the new typehint for the api?

Reply:
I left the docstring types for those types that caused recursive import problems [DEM-674], because Any is not very informative, but I deleted this now.